common/config/.golangci.yml

```diff
@@ -37,6 +37,7 @@ linters:
   enable:
   - deadcode
   - errcheck
+  - exportloopref
```

Comment:
this file is from istio/common-files, not istio/tools unfortunately. It is noted at the top of the file but extremely easy to miss.

Reply:
ha! I had those changes all ready to go, got distracted, and then came back a few hours later and picked the wrong VS window to commit and submit.